cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/StreamSizeScheduler.java

```diff
@@ -91,43 +196,116 @@ public void schedule(Id.Program program, SchedulableProgramType programType, Ite
   @Override
   public List<String> getScheduleIds(Id.Program program, SchedulableProgramType programType)
```

Comment:
Do you know what this method is good for? Nobody ever seems to call it. I am asking because I am wondering whether that caller would expect repeatable read?